lib/services/analytics/eqatec-analytics.ts

```diff
@@ -0,0 +1,52 @@
+import { AnalyticsServiceBase } from "../../common/services/analytics-service-base";
+
+export class EqatectAnalyticsService extends AnalyticsServiceBase {
+
+	constructor(private analyticsSettings: IEqatecInitializeData,
+		protected $logger: ILogger,
+		protected $staticConfig: Config.IStaticConfig,
+		$options: ICommonOptions,
+		$prompter: IPrompter,
+		$userSettingsService: UserSettings.IUserSettingsService,
+		$analyticsSettingsService: IAnalyticsSettingsService,
+		$progressIndicator: IProgressIndicator,
+		$osInfo: IOsInfo) {
+
+		super($logger, $options, $staticConfig, $prompter, $userSettingsService, $analyticsSettingsService, $progressIndicator, $osInfo);
+	}
+
+	public async trackInformation(data: IFeatureTrackingInformation): Promise<void> {
+		try {
+			await this.startEqatecMonitor(this.analyticsSettings);
+
+			if (this._eqatecMonitor) {
+				this._eqatecMonitor.trackFeature(`${data.featureName}.${data.featureValue}`);
+				await this.waitForSending();
+			}
+		} catch (e) {
+			this.$logger.trace("Analytics exception: '%s'", e.toString());
```

Comment:
I don't think `e.toString()` is relevant here - maybe just `this.$logger.trace("Analytics exception: ", e);`?